packages/components/src/components/progressbar/progressbar.component.ts

```diff
@@ -0,0 +1,143 @@
+ * @cssproperty --mdc-progressbar-label-fontsize - Font size of the label text.
+ * @cssproperty --mdc-progressbar-label-fontweight - Font weight of the label text.
+ * @cssproperty --mdc-progressbar-help-text-color - Color of the help text.
+ */
+class Progressbar extends DataAriaLabelMixin(FormfieldWrapper) {
+  /**
+   * Types of the progressbar
+   * - **Default**
+   * - **Inline**
+   *
+   * @default default
+   */
+  @property({ type: String, reflect: true })
+  variant: Variant = DEFAULTS.VARIANT;
+
+  /**
+   * The current progress as a percentage, 0 to 100.
+   * The value will be clamped between 0 and 100.
+   * @default 0
+   */
+  @property({ type: String, reflect: true })
+  value: string = '0';
+
+  /**
+   * Define error state of the progressbar
+   * - **true**
+   * - **false**
+   * @default false
+   */
+  @property({ type: Boolean, attribute: 'error' }) error = false;
+
+  constructor() {
+    super();
+    this.id = '';
+    this.disabled = undefined as unknown as boolean;
+  }
+
+  /**
+   * Ensures that the value is clamped between 0 and 100
+   * @returns The clamped value
+   */
+  private get clampedValue() {
+    const value = Number(this.value);
+    const clampedValue = Number.isNaN(value) ? 0 : Math.max(0, Math.min(100, value));
+    return clampedValue;
+  }
+
+  /**
+   * Determines the validation state (success, error, or default) based on progress value and error state.
+   * @returns The appropriate validation state for the progressbar.
+   */
+  private getValidationVariant() {
+    if (this.error) {
+      this.helpTextType = VALIDATION.ERROR;
+    } else if (this.clampedValue === 100) {
+      this.helpTextType = VALIDATION.SUCCESS;
+    } else {
+      this.helpTextType = VALIDATION.DEFAULT;
+    }
+    return this.helpTextType;
+  }
+
+  /**
+   * Renders the progress bar with dynamic width and variant styles.
+   * @returns The rendered HTML for the progress bar.
+   */
+  private renderProgressbar = () => {
+    const variant = this.getValidationVariant();
+    const isGap = this.clampedValue > 0 && this.clampedValue < 100;
+    const progressWidth = this.error ? '100' : `${this.clampedValue}`;
+    return html`
+      <div
+        part="progress-container ${isGap ? 'gap' : ''}"
+        role="progressbar"
+        aria-valuenow="${this.clampedValue}"
+        aria-valuemin="0"
+        aria-valuemax="100"
+        aria-label="${this.dataAriaLabel ?? ''}"
+      >
+        <div part="progress-bar ${variant}" style="width: ${progressWidth}%"></div>
+        <div part="remaining"></div>
+      </div>
+    `;
+  };
+
+  /**
+   * Renders the component based on the variant (inline or default).
+   * @returns The rendered HTML for the component.
+   */
+  public override render() {
+    const isInline = this.variant === VARIANT.INLINE;
+    return html`
+      ${isInline
+    ? html`<div part="inline-label-container">${this.renderLabel()} ${this.renderProgressbar()}</div>`
```

Comment:
```suggestion
    ? html`<div part="inline-label-container">${this.renderLabelElement()} ${this.renderProgressbar()}</div>`
```

Use renderLabelElement instead. This would only render the label text and no other slots.
renderLabel has slots for required label and icon info which is not required in this component.